### ISSUE TITLE
Documentation for sensor has the wrong platform and zone.home not found

### ISSUE THREAD

**Issue by dobrynski** [edekeijzer/osrm_travel_time#1]
The documentation lists `osrm` as the sensor platform but it is actually `osrm_travel_time`.

I am also getting the following error which doesn't seem to make sense:
```
2021-09-02 14:43:54 ERROR (MainThread) [custom_components.osrm_travel_time.sensor] Unable to find entity zone.home
2021-09-02 14:43:54 ERROR (MainThread) [custom_components.osrm_travel_time.sensor] Unable to find entity zone.home
```

Thanks in advance.

**Comment by edekeijzer** [comment]
You are right about the documentation, the `osrm` platform was changed when I published the integration to HACS. I'll update the documentation anytime soon.
Do you have a `zone.home` entity in your HA setup? Does it work when you specify latitude/longitude directly in the configuration?
